scripts/cocktails.js

```diff
@@ -0,0 +1,56 @@
+import {
+  getMetadata,
+  buildBlock,
+} from './lib-franklin.js';
+
+/**
+ * Loads the cocktail template and replaces the content when the metadata is set accordingly
+ * @param {Element} doc The container element
+ * @returns {Promise}
+ */
+// eslint-disable-next-line import/prefer-default-export
+export async function loadCocktail(doc) {
+  const templatePath = getMetadata('template');
+  if (templatePath !== '') {
+    // when the metadata define a template, retrieve it
+    const resp = await fetch(`${templatePath}.plain.html`, window.location.pathname.endsWith(`${templatePath}`) ? { cache: 'reload' } : {});
+
+    if (resp.ok) {
+      // load template content
+      const tplText = await resp.text();
+
+      // retrieve the content wrapper, our top most div and put it's cocktail content aside
+      const main = doc.querySelector('main');
+      const cocktailData = main.innerHTML;
```

Comment:
instead of serialization, we should use a faster approach.

Reply:
As discussed, it should work something like this:

```
diff --git a/scripts/cocktails.js b/scripts/cocktails.js
index c7b5414..813f029 100644
--- a/scripts/cocktails.js
+++ b/scripts/cocktails.js
@@ -21,14 +21,15 @@ export async function loadCocktail(doc) {
 
       // retrieve the content wrapper, our top most div and put it's cocktail content aside
       const main = doc.querySelector('main');
-      const cocktailData = main.innerHTML;
+      const cocktailData = Array.from(main.children);
 
       // apply the template content
       main.innerHTML = tplText;
 
       // reapply the cocktails content
       const cocktailDiv = main.querySelector('div.cocktail');
-      cocktailDiv.innerHTML = cocktailData;
+      cocktailDiv.textContent = '';
+      cocktailDiv.append(...cocktailData);
 
       const insertLocation = cocktailDiv.previousElementSibling;
```